src/_includes/macros/form.html

```diff
@@ -0,0 +1,244 @@
+{# ==========================================================================
+
+   form.input()
+
+   ==========================================================================
+
+   Description:
+
+   Builds form text field.
+
+   label: Name of the field.
+
+   type: Type of field. Will default to text, but can change to other
+         types to take advantage of HTML5 validation
+          Possible fields: text, tel, email, url, number, date.
+
+   size: The size of the field. Defaults to full-width.
+          Possible options: 1, 1-2, 1-3
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro input(label, type='text', size='1', required=false, disabled=false) -%}
+  {% from 'macros/util/format/url.html' import slugify as slugify %}
+  {% set id_slug = slugify( label ) %}
+  <div class="form-l_col form-l_col-{{ size }}">
+    <label class="form-label-header" for="form_{{ id_slug }}">
+      {{ label }}
+      {%- if required -%}
+        * <span class="u-visually-hidden">Required</span>
+      {%- endif -%}
+    </label>
+    <input class="input__long"
+           id="form_{{ id_slug }}"
+           type="{{ type }}"
+           {%- if required -%}required{%- endif -%}
+           {%- if disabled -%}disabled{%- endif -%}
+           >
+  </div>
+{%- endmacro %}
+
+{# ==========================================================================
+
+   form.textarea()
+
+   ==========================================================================
+
+   Description:
+
+   Builds form textarea field
+
+   label: Name of the field.
+
+   helperText:
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro textarea(label, helperText=false, required=false, disabled=false) -%}
+  {% from 'macros/util/format/url.html' import slugify as slugify %}
```

Comment:
I thought if someone wanted to import one of the macros it wouldn't work if the import wasn't inside, but I can test it to find out!